packages/core/test/lib/commands/debug.js

```diff
@@ -26,24 +27,26 @@ describe("debug", function () {
   });
 
   describe("mergeConfigNetwork(config, options)", function () {
+    const url = "http://urlhost:1234";
+    const parsedUrl = new URL(url);
+    const expectedNetworkName = parsedUrl.host;
+
     beforeEach(function () {
       config = Config.default();
-      options = {
-        url: "http://urlhost:1234"
-      };
+      options = { url: url };
     });
 
     it("should create networks item in config", function () {
       result = mergeConfigNetwork(config, options);
 
-      assert.notEqual(result.networks.inline_config, undefined);
-      assert.equal(result.networks.inline_config.url, "http://urlhost:1234");
-      assert.equal(result.networks.inline_config.network_id, "*");
+      assert.notEqual(result.networks[expectedNetworkName], undefined);
```

Comment:
More style notes - maybe we should get real nice with the chai assertions. Here we can use `assert.isDefined` https://www.chaijs.com/api/assert/#method_isdefined. There are a lot of really nice assertions in the chai docs there that we could probably use.